src/AzureStorageFS.ts

```
@@ -228,7 +230,7 @@ export class AzureStorageFS implements vscode.FileSystemProvider, vscode.TextDoc
                 result = await this.streamToString(downloaded.readableStreamBody);
             } catch (error) {
                 const pe = parseError(error);
-                if (pe.errorType === 'BlobNotFound' || pe.errorType === 'ResourceNotFound') {
+                if (pe.errorType === '404') {
```

Do we need to add a comment anywhere explaining what this is doing?

---

I mean, I feel like a 404 error is probably clear enough that we're just ignoring the error if the file wasn't found.

Also I realized that stat was being called constantly because we open the empty blob in the editor (also the reason we have the create/delete events).

---

Ok, this PR and linked issue has enough context if we need to understand this later on. LGTM